### DISCUSSION TITLE
Problem with login psa car controller

### DISCUSSION THREAD

**Discussion by 500Cacique** [flobz/psa_car_controller#423] [Q&A]
Hi, I hace installed psa car controller as instructions of globs but when I compile the data user config, the result  is " 'NoneType' object is not subscriptable"
I had psa car controller before in HA but when I install a old backup I lost all of this configuration. 
I reinstalled the add on but with the same answer.
Could someone help me to solve it?
Thanks for your help

**Comment by kurim** [comment]
Did you use 2.x?
If yes please use 3.x from here: https://github.com/flobz/psacc-ha-beta
Only thing is you can't use ingress but you can complete the configuration.

**Reply by 500Cacique** [reply]
Hi, finally I install 3.0.5 version and I connect via web but when I try  to connect via web due to link of PSA I integration,  I only obtain the message  " loading..." but don't  finish. Do you know what is the  problem? Thanks for your help

**Comment by 500Cacique** [comment]
Hi, Félix, I use version 3.0.5. I could ingress but when  I tray click in
web bottom, the process remaining "loading".
Do you know what is the possible problem?
Thanks

**Reply by kurim** [reply]
That is the known issue:
https://github.com/flobz/psa_car_controller/issues/363#issuecomment-1265299240

You only can use homeassistant:5000 if enabled port 5000 in config
